ttnn/cpp/ttnn/operations/conv2d/conv2d.cpp

```diff
@@ -72,15 +72,24 @@ ParallelConfig determine_parallel_config(
     uint32_t max_num_cores = device_grid_size[0] * device_grid_size[1];
 
     auto calculate_num_cores_nhw = [&]() {
-        uint32_t num_cores_nhw =
-            height_sharding
-                ? find_closest_largest_divisor(conv_out_2d_matrix_height_ntiles, max_num_cores)
-                : find_closest_largest_divisor_with_num_padding(conv_out_2d_matrix_height_ntiles, device_grid_size[0]);
-        return num_cores_nhw;
+        if(conv_config.is_height_sharded())
+        {
```

Comment:
format as:
```
if () {
...
} else {
...
}
```

Reply:
As a I've mentioned before this applies to whole PR, go through your changes one more time and fix up this things